kegtab/build.gradle

```diff
@@ -111,11 +109,6 @@ dependencies {
     implementation 'joda-time:joda-time:2.3'
     implementation 'com.android.support:multidex:1.0.3'
 
-
-    implementation('com.crashlytics.sdk.android:crashlytics:2.10.1@aar') {
```

Comment:
Removed Crashlytics entirely as this no longer is supported and we eventually need to migrate to Firebase Crashlytics.

@mik3y Is this something that is even used anymore?  I noticed that the APK on the Play Store hasn't been updated in years.